src/styles/blocks/page.scss

```diff
@@ -0,0 +1,44 @@
+@import "utils/variables.scss";
+@import "utils/mixins.scss";
+
+.page {
+  font-family: Poppins, sans-serif;
+  font-size: 16px;
+  color: $color-white;
+  scroll-behavior: smooth;
+  overflow-x: hidden;
+
+  &--menu-open {
+    overflow: hidden;
+  }
+
+  &__body {
```

Comment:
As mentioned previously, by reading the scss alone I immediately assume this styles body element